features/adding_product_to_selected_wishlist.feature

```diff
@@ -0,0 +1,18 @@
+@wishlist
+Feature: Adding a product to selected wishlist
+    In order to compare or buy products later
+    As a Visitor
+    I want to be able to add products to one of my wishlists
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @ui
+    Scenario: Adding a product to selected wishlist
+        Given I am on "/"
```

Comment:
When we have only one scenario, all given's should be put into background I think